### ISSUE TITLE
rubyLocate not working correctly

### ISSUE THREAD

**Issue by archfish** [rubyide/vscode-ruby#385]
### My environment

- `vscode-ruby` version: 0.20.0 
- Ruby version: 2.1.9
- Ruby version manager (if any): rvm
- VS Code version:   1.27.1
- Operating System: MacOS 10.13.6
- Using language server? NO

### Expected behavior

ruby intellisense with rubyLocate using in workspace should detected `locate` by current project not `vscode.workspace.rootPath`.

### Actual behavior

rubyLocate using first project path as rootPath

*This and the next section should include screenshots, code samples, console output, etc. The more information we have to reproduce the better!*

```js
function registerIntellisenseProvider(ctx) {
  ...
  let locate = new locate_1.Locate(vscode.workspace.rootPath, settings);
  console.warn(`vscode.workspace.rootPath: ${vscode.workspace.rootPath}`)
  ...
}
```

got 

```log
vscode.workspace.rootPath: /Users/mymac/GoPath/src/github.com/fagongzi/gateway
```

my workspace config:

```json
{
	"folders": [
		{
			"path": "/Users/mymac/GoPath/src/github.com/fagongzi/gateway"
		},
		{
			"path": "reocar_store"
		},
	],
        "settings": {
        }
}
```

gateway is golang project and reocar_store is an ruby on rails project. 

when move reocar_store to first project of workspace it work perfect.

**Comment by wingrunr21** [comment]
This extension does not currently support multi-root workspaces. Support for that is in progress but there's no ETA

**Comment by selialkile** [comment]
@wingrunr21 We can do a fix around creating a provider or class to deal with this many roots, sending all events of modified files to that new class, what you think?

**Comment by wingrunr21** [comment]
Multi-root support was merged in #405. However, I will not be moving rubyLocate support to the language server as I will be utilizing the existing AST infrastructure there to facilitate Intellisense features
